packages/flutter/lib/src/cassowary/equation_member.dart

```diff
@@ -5,25 +5,95 @@
 import 'expression.dart';
 import 'constraint.dart';
 
-/// Base class for the various parts of cassowary equations.
+/// A member that can be used to construct an [Expression] that may be
+/// used to create a constraint. This is to facilitate the easy creation of
+/// constraints. The use of the operator overloads is completely optional and
+/// is only meant as a convenience. The [Constraint] expressions can be created
+/// by manually creating instance of [Constraint] variables, then terms and
+/// combining those to create expression.
 abstract class EquationMember {
+  /// The representation of this member after it is hoisted to be an
+  /// expression.
   Expression asExpression();
 
+  /// Returns if this member is a constant. Constant members can be combined
+  /// more easily without making the expression non-linear. This makes them
+  /// easier to use with multiplication and division operators. Constant
+  /// expression that have zero value may also eliminate other expressions from
+  /// the solver when used with the multiplication operator.
   bool get isConstant;
 
+  /// The current constant value of this member. After a [Solver] flush, this is
+  /// value read by entities outside the [Solver].
   double get value;
 
+  /// Creates a [Constraint] by using this member as the left hand side
+  /// expression and the argument as the right hand side [Expression] of a
+  /// [Constraint] with a [Relation.greaterThanOrEqualTo] relationship between
+  /// the two.
+  ///
+  /// For Example: `right - left >= cm(200.0)` would read, "the width of the
+  /// object is at least 200."
   Constraint operator >=(EquationMember m) => asExpression() >= m;
 
+  /// Creates a [Constraint] by using this member as the left hand side
+  /// expression and the argument as the right hand side [Expression] of a
+  /// [Constraint] with a [Relation.lessThanOrEqualTo] relationship between the
+  /// two.
+  ///
+  /// For Example: `rightEdgeOfA <= leftEdgeOfB` would read, "the entites A and
+  /// B are stacked left to right."
   Constraint operator <=(EquationMember m) => asExpression() <= m;
 
+  /// Creates a [Constraint] by using this member as the left hand side
+  /// expression and the argument as the right hand side [Expression] of a
+  /// [Constraint] with a [Relation.equalTo] relationship between the two.
+  ///
+  /// For Example: `topEdgeOfBoxA + cm(10.0) == topEdgeOfBoxB` woud read,
```

Comment:
ditto

there are other "For Example"s that should be "For example", i won't point them all out